### PR TITLE
updates node-observability Operator using a custom index image on OperatorHub procedure

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
@lmzuccarelli Please find my suggestions and let me know your feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,47 +36,46 @@ You can install the NodeObservability Operator by building and pushing the Opera
    ```
 
 
-### Installing the `Node Observability` Operator using a custom index image on OperatorHub
-**Note**: The below procedure works best with `podman` as container engine
-
```
Comment (darshan-nagaraj) [inline README.md:41]: @lmzuccarelli Please share the prerequisites required to carry out this procedure.

File: README.md [hunk 2]
```diff
@@ -108,8 +107,8 @@ You can install the NodeObservability Operator by building and pushing the Opera
     EOF
     ```
 
-9. Create the `Subscription` object:
-    ```sh
+8. Create the `Subscription` object:
+    ```
     cat <<EOF | oc apply -f -
     apiVersion: operators.coreos.com/v1alpha1
     kind: Subscription
```
Comment (darshan-nagaraj) [inline README.md:114]: @lmzuccarelli Is there any verification steps available post installation ? Please share

**Review by darshan-nagaraj — COMMENTED** [review]
I have added few comments along with my PR. Please review and let me know your feedback.

**Review by lmzuccarelli — COMMENTED** [review]
@darshan-nagaraj - This looks good. Thanks for the help on this PR. I will add a the looks good to me label

File: README.md (hunk 1 above)
Comment (darshan-nagaraj) [reply README.md:41]: Done.

**Pull request merged**